for_developers/doxygen/index.md

```diff
@@ -32,6 +32,7 @@ browser, the file `${HOME}/rootdoc/html/index.html`.
 ### Compilation prerequisites on Ubuntu
 - apt install doxygen make jupyter-nbconvert 
 - pip install scandir nbformat metakernel
+- python2 -m pip install ipykernel
```

Comment:
uhm, shouldn't python3 be the default these days?

Reply:
Hmmm the Makefile is currently not forcing the user to have a minimum python version. There is now:
`Makefile:PYTHON_EXECUTABLE ?= /usr/bin/python`

The root make-doc build machine has /usr/bin/python pointing to python2, as compared to /usr/bin/python3, so couet found that installing this extra package solved the many errors that were being found in the console log https://lcgapp-services.cern.ch/root-jenkins/job/root-makedoc-master/2651/consoleFull jupyter_client.kernelspec.NoSuchKernel: No such kernel named python2

(That being said, it would be also fine for me to change the Python_executable in the Makefile to python3).

Reply:
i can't comment as to why the machine where we build the docs uses python 2, but most developers and users these days are on python 3 (and they should be, python 2 is dead since january 2020). so when explaining to new devs how to generate the doxygen docs, we should not impose python 2 -- we can just write `python` here.

Reply:
Good point. I merged it with the line above.